lib/ensure_content_type.js

```diff
@@ -78,18 +78,34 @@ function checkContentType(hyper, req, next, expectedContentType, responsePromise
 }
 
 module.exports = (hyper, req, next, options, specInfo) => {
-    const produces = specInfo.spec.produces;
```

Comment:
Omg, this is something that has been missed entirely when updating to openapi 3. Thank you for noticing and fixing it.